core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/StackToCloudStackConverter.java

```diff
@@ -395,6 +407,28 @@ private SpiFileSystem buildAdditionalSpiFileSystem(Stack stack) {
         return spiFileSystem;
     }
 
+    public GroupNetwork buildGroupNetwork(com.sequenceiq.cloudbreak.domain.Network stackNetwork, InstanceGroup instanceGroup) {
+        GroupNetwork groupNetwork = null;
+        InstanceGroupNetwork instanceGroupNetwork = instanceGroup.getInstanceGroupNetwork();
+        if (instanceGroupNetwork != null) {
+            Json attributes = instanceGroupNetwork.getAttributes();
+            Map<String, Object> params = attributes == null ? Collections.emptyMap() : attributes.getMap();
+            Set<GroupSubnet> subnets = new HashSet<>();
+            if (params != null) {
+                List<String> subnetIds = (List<String>) params.getOrDefault(NetworkConstants.SUBNET_IDS, new ArrayList<>());
```

Comment:
Nit: What is the benefit of `getOrDefault()` vs a simple `get()` if we are checking for `subnetIds != null` anyway in the next line?

Reply:
removed null check